identity/handler.go

```diff
@@ -106,23 +113,23 @@ type adminListIdentities struct {
 
 // swagger:route GET /identities v0alpha2 adminListIdentities
 //
-// List Identities
+// # List Identities
 //
 // Lists all identities. Does not support search at the moment.
 //
 // Learn how identities work in [Ory Kratos' User And Identity Model Documentation](https://www.ory.sh/docs/next/kratos/concepts/identity-user-model).
 //
-//     Produces:
```

Comment:
Are you sure about these comments' changes in this file? They look Ory's. If so, I wouldn't touch it to avoid conflicts on merges in the future.

Reply:
But, if it's because of some inconsistency in the file, then it makes sense to propose these to the upstream.

Reply:
ah, I've removed them from this endpoint by accident, thanks

Reply:
looks like IDE is formatting it, I think we can keep it and I can remove it if upstream also wants me to do it